pages/content/closed-captioning.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: primary
+title: Video captioning
+permalink: /content-design/video-captioning/
+category: Content design
+sidebar: content
+question: Is everyone able to access your multimedia content?
+personas:
+- Blake is hearing-impaired and cannot rely on audio.
+- Sandra is a non-native English speaker and has difficulty understanding video.
+---
+
+### Steps to take
```

Comment:
We don't talk about making sure we have a transcript.

Reply:
Were you saying that I haven't mentioned a case in which someone may need a transcript? Or were you pointing out the case of Sandra would not be applicable here? I wasn't too clear.

Reply:
Since the comment was on "Steps to take," my guess is he meant the former.